src_python/habitat_sim/utils/settings.py

```diff
@@ -32,6 +32,7 @@
     "seed": 1,
     "physics_config_file": "data/default.physics_config.json",
     "enable_physics": True,
+    "pbr_image_based_lighting": False,
```

Comment:
Will there be other types of lighting supported in the future, should we make this an enum to future proof the API?

Reply:
Not exclusionary, to the best of my knowledge.  We will also add an HBAO flag eventually.

Reply:
Actually I did set up the fragment shader to support picking which extensions are supported (to isolate potential bugs and also to possibly speed up rendering), but that's in the most rudimentary stage at the moment, and I'm not sure how the functionality will be exposed to the users.  A bit early to add anything related to that in this PR.